src/ngraph/op/util/attr_types.hpp

```diff
@@ -49,5 +51,28 @@ namespace ngraph
             AUTO = SAME_UPPER,
             NOTSET = EXPLICIT
         };
+
+        enum class AutoBcastType
```

Comment:
Would prefer to change this to `AutoBroadcastType`

Reply:
you really dont like Bcast :)

Reply:
I just really like typing.